src/panels/lovelace/cards/hui-area-card.ts

```diff
@@ -72,6 +73,7 @@ const DOMAIN_ICONS = {
   sensor: { humidity: mdiWaterPercent },
   binary_sensor: {
     motion: mdiRun,
+    moisture: mdiWaves,
```

Comment:
I think `water-alert` would make more sense here

Reply:
Makes sense, updated. This is what it looks like now:

![2022-06-22-210121_910x525_scrot](https://user-images.githubusercontent.com/117643/175125919-aa264d6f-54ae-42e5-9d8c-bb836bc90d37.png)